README.md

```diff
@@ -77,6 +79,56 @@ git submodule add https://github.com/greenfork/jzignet libpath
 const jzignet = @import("jzignet");
 ```
 
+## How to include as dependency (with zon)
+
+This method of including jzignet in your project is more complicated.
+
+First, create a file named `build.zig.zon` with the following content. Replace `<commit>` in the file with the git commit you want to use, for example, `943ace7`.
+
+```zig
+.{
+    .name = "hello-world",
+    .version = "0.0.1",
+    .dependencies = .{
+        .jzignet = .{
+	        .url = "https://github.com/greenfork/jzignet/archive/<commit>.tar.gz",
+        },
+    },
+}
+```
+
+Next, run `zig build`. There should be an error telling you to include `.hash = ......` in `build.zig.zon`. Paste the line from the terminal.
```

Comment:
Is this instruction for me or for the end user? I have a feeling that that I need to do this manually and provide a hash inside the readme for the tarball I have created

Reply:
You can put this code snippet on each release page, with the correct hash.